demisto_sdk/commands/content_graph/common.py

```diff
@@ -0,0 +1,169 @@
+
+import enum
+import os
+from pathlib import Path
+from typing import Any, Dict, Iterator, List, Set
+from demisto_sdk.commands.common.tools import get_content_path
+
+REPO_PATH = Path(get_content_path())
+
+NEO4J_ADMIN_DOCKER = ''
+
+NEO4J_DATABASE_URL = os.getenv('DEMISTO_SDK_NEO4J_DATABASE_URL', 'bolt://localhost:7687')
```

Comment:
These 3 default values should be removed, unless they're NEO4J's default values.

Reply:
Only the password is not a default value, but we use this password to access the local DB (wheter in docker or locally)
The user can configure an instance with cloud as well